src/views/Languages/index.jsx

```diff
@@ -0,0 +1,297 @@
+          })),
+          'summary'
+        )) ||
+      [];
+    const mentoredBugs =
+      (bugzillaData &&
+        bugzillaData.mentored &&
+        uniqBy(
+          bugzillaData.mentored.edges.map(edge => edge.node).map(bug => ({
+            assignee: bug.status === 'ASSIGNED' ? bug.assignedTo.name : '-',
+            project: bug.component,
+            tags: [
+              ...(bug.keywords || []),
+              ...extractWhiteboardTags(bug.whiteboard),
+            ],
+            summary: bug.summary,
+            lastUpdated: bug.lastChanged,
+            id: bug.id,
+            url: `https://bugzilla.mozilla.org/show_bug.cgi?id=${bug.id}`,
+          })),
+          'summary'
+        )) ||
+      [];
+
+    return (
+      <div className={classes.root}>
+        <AppBar position="fixed" className={classes.header}>
+          <Grid
+            container
+            className={classes.title}
+            alignItems="center"
+            spacing={8}>
+            <Hidden mdUp>
+              <Grid item>
+                <IconButton
+                  className={classes.button}
+                  size="large"
+                  onClick={this.handleDrawerToggle}>
+                  <MenuIcon />
+                </IconButton>
+              </Grid>
+            </Hidden>
+            <Grid item>
+              <Typography
+                align="center"
+                variant="display1"
+                noWrap
+                className={classes.link}
+                component={Link}
+                to="/">
+                Codetribute
+              </Typography>
+            </Grid>
+          </Grid>
+        </AppBar>
+        <Hidden mdUp>
+          <Drawer
+            variant="temporary"
+            anchor="left"
+            open={drawerOpen}
+            onClose={this.handleDrawerToggle}
+            classes={{
+              paper: classes.drawerPaper,
+            }}
+            ModalProps={{
+              keepMounted: true,
+            }}>
+            {drawer}
+          </Drawer>
+        </Hidden>
+        <Hidden smDown implementation="css">
+          <Drawer
+            variant="permanent"
+            open
+            PaperProps={{
+              elevation: 2,
+            }}
+            classes={{
+              paper: classes.drawerPaper,
+            }}>
+            {drawer}
+          </Drawer>
+        </Hidden>
+        <div className={classes.container}>
+          {error && <ErrorPanel error={error} />}
+          {bugzillaData &&
+            bugzillaData.error && <ErrorPanel error={bugzillaData.error} />}
+          {bugzillaData &&
+            bugzillaData.loading && <Spinner className={classes.spinner} />}
+          {(!bugzillaData || !bugzillaData.loading) && (
+            <TasksTable
+              items={[...goodFirstBugs, ...mentoredBugs]}
+              onBugInfoClick={this.handleBugInfoClick}
+            />
+          )}
+        </div>
+      </div>
+    );
+  }
+}
```

Comment:
Instead of duplicating the Drawer, could we create a Dashboard component (somewhat similar to tc-web) that takes care of rendering `AppBar` and `Drawer`? We can have this component expose a `withSkills` prop defaulting to `false`. When `withSkills` is set to `true`, the drawer can be displayed. As a result, we won't need to duplicate all of the drawer and Appbar CSS.